apps/gk_neut_species.c

```diff
@@ -16,7 +16,7 @@
 void
 gk_neut_species_init(struct gkyl_gk *gk, struct gkyl_gyrokinetic_app *app, struct gk_neut_species *s)
 {
-  int cdim = app->cdim, vdim = app->vdim+1; // neutral species are 3v
+    int cdim = app->cdim, vdim = app->vdim+1; // neutral species are 3v
```

Comment:
new indent

Reply:
Fixed in [54e6fc0](https://github.com/ammarhakim/gkylzero/pull/567/commits/54e6fc0d7589e76cd30a6662b7a3e61df38e518e)